specs/latest/1.0/index.html

```diff
@@ -395,6 +395,54 @@ <h3><a name="THE_DRAWING_BUFFER">The Drawing Buffer</a></h3>
         </p>
     </div>
 
+    <dl class="methods">
+        <dt class="idl-code">undefined backbufferStorage(GLenum sizedFormat, unsigned long width, unsigned long height);
+        <dd>
+            Respecify the size and format of the backbuffer,
+            similar to <code>renderbufferStorage</code> as used on Renderbuffers.
+
+            Clearing behavior is equivalent to setting <code>HTMLCanvasElement.width</code> and <code>HTMLCanvasElement.height</code>, followed by changing the backbuffer format.
```

Comment:
If the user first calls `renderBufferStorage` and subsequently sets the `HTMLCanvasElement.width` and `HTMLCanvasElement.height` attributes, will the default framebuffer be resized to the last format passed as the `sizedFormat` parameter?  Or will the default framebuffer be resized to whatever the default format the user agent decided when it first allocated the default framebuffer?  The former seem preferable but I don't have a strong opinion.

Reply:
It'll use the last format passed in to `backBufferStorage` - which per the last WebGL WG meeting will be renamed to `drawingBufferStorage` in the next iteration of this proposal.